FAQ.md

```diff
@@ -56,6 +56,17 @@ git config --global --add safe.directory /path/to/repository
 
 ```
 
+You can learn more about diff-aware scanning in our [dedicated documentation](doc/diff-aware.md).
+
+
+## I have a bug to report
+
+Please send us a bug report, either by [creating an issue](https://github.com/DataDog/datadog-static-analyzer/issues) 
```

Comment:
"Please [create a GitHub issue] (https://github.com/DataDog/datadog-static-analyzer/issues) or [contact Datadog support](https://www.datadoghq.com/support/)."

Not all customers will have a dedicated CSM.